package.json

```diff
@@ -80,15 +80,15 @@
     "zone.js": "~0.10.2"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.900.4",
-    "@angular-devkit/build-ng-packagr": "^0.900.4",
-    "@angular/cli": "^9.0.5",
-    "@angular/compiler": "^9.0.6",
-    "@angular/compiler-cli": "^9.0.6",
-    "@angular/language-service": "^9.0.6",
-    "@angular/platform-browser-dynamic": "^9.0.6",
-    "@angular/platform-server": "^9.0.6",
-    "@angular/router": "^9.0.6",
+    "@angular-devkit/build-angular": "^0.901.0",
+    "@angular-devkit/build-ng-packagr": "^0.901.0",
+    "@angular/cli": "^9.1.0",
+    "@angular/compiler": "^9.1.0",
+    "@angular/compiler-cli": "^9.1.0",
+    "@angular/language-service": "^9.1.0",
+    "@angular/platform-browser-dynamic": "^9.1.0",
+    "@angular/platform-server": "^9.1.0",
+    "@angular/router": "^9.1.0",
     "@commitlint/cli": "^8.2.0",
     "@nguniversal/builders": "^9.0.0",
```

Comment:
should we upgrade this one as well, or doe we leave it in #811

Reply:
Thanks, i missed that one, and upgraded it :)